darts/tests/explainability/test_shap_explainer.py

```diff
@@ -377,7 +376,7 @@ def test_explain_with_lags_future_covariates_series_of_same_length_as_target(sel
         model = LightGBMModel(
             lags=4,
             lags_past_covariates=[-1, -2, -3],
-            lags_future_covariates=[2],
```

Comment:
Right, that makes sense. I guess I was just coming from more of a causal inference perspective, where its generally accepted that future values can't 'cause' past values, but I guess that isn't necessarily true if one were to 'know' the future (since they'd be able to act on this information in the present). So in summary, I think you're right about this - we should allow future lags to be positive.

Reply:
Exactly, for instance knowing future weather forecasts may prompt different actions and have a causal impact on targets.